packages/react-table/single-packages.config.json

```diff
@@ -2,16 +2,8 @@
     "packageName": "@patternfly/react-table",
     "modules": [ 
         {
-            "name": "components",
+            "name": "dist/esm",
             "type": "esm"
-        },
-        {
-            "name": "deprecated",
```

Comment:
Are deprecated components are still being delivered via these federated modules? They will need to be

Reply:
I have a call with @dlabaj where I want to clear up some questions. The deprecated modules is one of them.

Reply:
They are the build script has been updated to point to the deprecated components.  In the dynamic folder it contains all the esm versions for the deprecated components.

Reply:
Yeah, we no longer need separate builds. All of the files are emitted during a single cycle now.